client/asset/btc/btc.go

```diff
@@ -2365,6 +2377,209 @@ func (btc *baseWallet) fund(keep uint64, minConfs uint32, lockUnspents bool,
 	return btc.fundInternal(keep, minConfs, lockUnspents, enough)
 }
 
+// fundMultiBestEffors makes a best effort to fund every order. If it is not
+// possible, it returns coins for the orders that could be funded. The coins
+// that fund each order are returned in the same order as the values that were
+// passed in.
```

Comment:
They're not really requesting more than is available, it's just that without a split the wallet would not be able to fund all of the orders without going over the `keep` amount.

Reply:
If the caller asks to fund 5 orders, but we return partial results for 3, is the caller going to place those 3 orders? Do they have any say in which orders get funded?

It's kinda confusing because for normal order funding, if we can't do it without a split, we definitely can't do it with a split. I don't think that's the case with `FundMulti`, because overlocks stack.

Reply:
I had roughly the same question: https://github.com/decred/dcrdex/pull/2362#discussion_r1210727916
It stops on the first one that fails.  I agree that's a difficult method to use effectively.  I think it would need a flag indicate if you want to allow that, or return the coins and fail entirely if you can't get them all.

Related, in caller, it returns and error even if some orders were submitted: https://github.com/decred/dcrdex/pull/2362#discussion_r1212192557

Reply:
The idea was that the caller passes the orders in order of priority. I can definitely add a flag. I was considering adding an `allOrNothing` flag for this, but I didn't because I didn't see any of the bots passing true for this.